packages/core-components/src/components/button/button.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../../odyssey/src/scss/base/reset';
+@import '../../../../odyssey/src/scss/base/typography-link';
+@import '../../../../odyssey/src/scss/components/button';
```

Comment:
We shouldn't link directly to the Odyssey source here. Instead, let's import it as we would any CSS project.